configuration/pih/htmlforms/ncd-adult-followup.xml

```diff
@@ -8,20 +8,6 @@
     <postSubmissionAction class="org.openmrs.module.pihcore.htmlformentry.action.CleanDiagnosisConstructAction"/>
     <postSubmissionAction class="org.openmrs.module.pihcore.htmlformentry.action.CleanPrescriptionConstructAction"/>
 
-    <macros>
```

Comment:
The key change to this file is that everywhere that used to be wrapped in a macro conditional is now wrapped in an html element with a class.  Javascript queries the patient's current conditions as of the encounter date, and shows/hides sections for ncds based on this.